src/style.css

```diff
@@ -0,0 +1,63 @@
+.styleDefault {
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 px 1px 8px /rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+  /* line-height: 22px; */
+}
+
+.styleDefault1 {
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 px 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  width: 100%;
```

Comment:
don't repeat yourself, create styles according to 2 point in the checklist

Reply:
- [DEMO LINK](https://Ivankaaaaaa.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://Ivankaaaaaa.github.io/layout_search-bar-airbnb/report/html_report/)